src/main/java/org/reaktivity/nukleus/http/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -417,7 +476,10 @@ private void processReset(
             int length)
         {
             resetRO.wrap(buffer, index, index + length);
-
+            if (slotIndex != SLOT_NOT_AVAILABLE)
+            {
+                slab.release(slotIndex);
+            }
```

Comment:
Would it be easier to manage this `if` condition inside `slab.release(...)` method instead of at each call site?